packages/babel-preset-gatsby-package/src/__tests__/index.js

```diff
@@ -0,0 +1,109 @@
+const preset = require(`../`)
+
+it(`Specifies proper presets and plugins in Node mode`, () => {
+  const { presets, plugins } = preset()
+
+  expect(presets).toEqual([
+    [
+      expect.stringContaining(`@babel/preset-env`),
```

Comment:
This won't work on Windows (😅). I'll test it like the babel helpers are tested right now by also resolving the module in the test and comparing the resolved paths instead

Reply:
Windows is a constant pain 🙃 I think once this is resolved we can get this merged 🎉